site/class-1-load/Dorusth/script.js

```diff
@@ -0,0 +1,27 @@
+//inladen van de csv
```

Comment:
You should name your javascript `index.js` instead of `script.js`.

You can remove files through the github interface, by going to your fork, your branch, and the finding your file. Finally, click on the trash icon in the top-right.